lib/super_good/solidus_taxjar/testing_support/factories/order_transaction_factory.rb

```diff
@@ -3,12 +3,20 @@
     order
     transaction_date { Date.current }
 
-    sequence(:transaction_id) { |n|
-      if n == 1
-        order.number
-      else
-        "#{order.number}-#{n - 1}"
-      end
+    transient do
+      last_transaction_id {
```

Comment:
Do we really care about what transaction id's the order factory generates 🤔 If we need to test against that in tests we should pass in the value rather than relying on the factory behaving like the application code?

I am not really keen on re-implementing this in the factory itself as factories are meant to just provide a valid fake record.

Reply:
I think that it is safe for this factory to change as the application code changes.

That way, we are always at least generating very-realistic factory objects.